oak_functions/loader/src/main.rs

```diff
@@ -48,48 +53,57 @@ pub struct Opt {
         help = "Path to a Wasm file to be loaded and executed per invocation. The Wasm module must export a function named `main`."
     )]
     wasm_path: String,
-    // TODO(#1930): Support downloading lookup data from a URL instead of a local file.
     // TODO(#1930): Support periodically re-downloading lookup data at a given time interval.
     #[structopt(
         long,
-        help = "Path to a file to load for data lookups, containing key / value entries in protobuf binary format."
+        help = "URL to a file to GET over HTTP containing key / value entries in protobuf binary format for lookup."
     )]
-    lookup_data_path: String,
+    lookup_data_url: String,
 }
 
 struct LookupData {
-    lookup_data_path: String,
+    lookup_data_url: String,
```

Comment:
Is this data loaded once, or should it be loaded once per interval, to update the database?

Reply:
There is no interval yet, just a TODO to implement the periodic refresh for now.